app/models/concerns/system_like.rb

```diff
@@ -48,7 +48,15 @@ def rules_failed(profile = nil)
   end
 
   def compliance_score
-    score = (100 * (rules_passed.to_f / (rules_passed + rules_failed)))
+    counts = test_results.latest.joins(:rule_results)
```

Comment:
This bypasses `last_scanned` method and the context (`profile_id`) that reduces the scope of results for score count. I wonder if this would have a consequence, maybe not.

`rules_passed` and `rules_failed` are memoized on its model and are contextful. Now it depends if that context is before or after the models are constructed.

It would be good to verify this with a test.

Reply:
This PR will be obsolete if #915 gets merged